content/concepts/asset-manager.md

```diff
@@ -96,6 +96,121 @@ Here is the JSON format representing an example of asset family.
+        "locale": null
+    },
+    "pattern": "/(?P<product_ref>.*)-.*-(?P<attribute_ref>.*)\.jpg/",
+    "abort_asset_creation_on_error": true
+  }
+}
+```
+
+Still not comfortable with the naming convention? Don't hesitate to go through the next sections where we detail each part of the naming convention format.
+
+### The source string
+
+The `source` property allows you to define on which string the split will be applied. It can be either:
+- the asset code,
+- the code of the main media attribute of your family.
+
+It follows this format:
+```json
+{
+  "source": {
+    "property": CODE_OR_ATTRIBUTE_CODE,
+    "locale": SOURCE_LOCALE_CODE,
+    "channel": SOURCE_CHANNEL_CODE
+  },...
+}
+```
+
+In this formula:
+ - `CODE_OR_ATTRIBUTE_CODE` can be either:
+     + the _"code"_ string: when you want the asset code to be used as the source string,
+     + the code of the main media asset attribute of the family: when you want the filename of the main media to be used as the source string.
+ - `SOURCE_LOCALE_CODE` is an existing locale code when `CODE_OR_ATTRIBUTE_CODE` is equal to _"main_media"_ and the main media attribute of the family is localizable.
+ - `SOURCE_CHANNEL_CODE` is an existing channel code when `CODE_OR_ATTRIBUTE_CODE` is equal to _"main_media"_ and the main media attribute of the family is scopable.
+
+::: warning
+The `property`, `locale` and `channel` properties are mandatory.  
+The `locale` property should be set to `null` if:
+- `CODE_OR_ATTRIBUTE_CODE` is equal to _"code"_,
+- `CODE_OR_ATTRIBUTE_CODE` is equal to the code of the main media attribute and this one is not localizable.  
+The `channel` property should be set to `null` if:
+- `CODE_OR_ATTRIBUTE_CODE` is equal to _"code"_,
+- `CODE_OR_ATTRIBUTE_CODE` is equal to the code of the main media attribute and this one is not scopable.  
+:::
+
+### The split pattern
+
+The `pattern` property allows you to define how the PIM should split the [source string](#the-source-string). Then, the result of the split will automatically populate the corresponding asset attributes.
+
+The split pattern should be a string. It should be given as a regular expression.  
+In order for the PIM to know into which asset attributes the result of the split should be sent, this regular expression should contain one or several named capture groups and the names of these capture groups should be existing asset attribute codes of the family.
+
+::: tips
+Not comfortable with regular expressions? You can try yours [right here](https://regex101.com/)!
+:::
+
+Let's take an example to make this clearer! 
+```regexp
+/(?P<product_ref>.*)-.*-(?P<attribute_ref>.*)\.jpg/
+```
+The regexp above will split the source string in 3 parts, thanks to 2 named capture groups:
+- `(?P<product_ref>.*)` is the first capture group. It is named `product_ref`. So, the result of this capture will be sent into the `product_ref` asset attribute. The `product_ref` attribute should exist in the asset family.
```

Comment:
Should we say that attributes should be not localizable neither scopable?

Reply:
Yas, definitely!

Reply:
Done!